types/client.d.ts

```diff
@@ -9,7 +9,7 @@ declare module 'client' {
     clean: Readonly<boolean>
     version: Readonly<number>
     conn: Connection
-    req?: IncomingMessage
+    req?: IncomingMessage | null
```

Comment:
```suggestion
    req?: IncomingMessage
```

Reply:
this change is required based on the changes required in the javascript code.

Reply:
See previous answer